qc-runner/src/numGenesVsNumUmis.r

```diff
@@ -33,7 +33,8 @@ source('utils.r')
 #'                                - p.level: which refers to  confidence level for deviation from the main trend
 #' @export return a list with the filtered seurat object by numGenesVsNumUmis, the config and the plot values
 
-task <- function(seurat_obj, config, task_name, sample_id, num_cells_to_downsample = 5000, percent_downsample = 20){
+
+task <- function(seurat_obj, config, task_name, sample_id, num_cells_to_downsample = 5000){
```

Comment:
```suggestion
task <- function(seurat_obj, config, task_name, sample_id, num_cells_to_downsample = 6000){
```
Shoudn't this be also 6000 as the others?

Reply:
Good catch. Not sure why this one is set to 5000. Maybe @JuanluOnieva-biomage or @aerlaut knows?

Reply:
Certainly looks like a typo

Reply:
I changed it to `6000` unless we hear otherwise